requirements.in

```diff
@@ -268,14 +269,28 @@ toolz==0.11.1
     #   partd
 tornado==6.1
     # via distributed
-typed-ast==1.4.3
-    # via mypy
+types-futures==0.1.3
+    # via mozilla-jetstream
+types-pkg-resources==0.1.2
+    # via mozilla-jetstream
+types-protobuf==0.1.12
+    # via mozilla-jetstream
+types-pytz==0.1.1
+    # via mozilla-jetstream
+types-PyYAML==0.1.9
+    # via mozilla-jetstream
+types-requests==0.1.11
+    # via mozilla-jetstream
+types-six==0.1.6
+    # via mozilla-jetstream
+types-toml==0.1.2
+    # via mozilla-jetstream
```

Comment:
There is also `mypy --install-types` but it requires an interactive `y` keyboard input. Installing the packages directly seems to be the best option here